packages/datatrak-web-server/src/routes/RecentSurveyResponseRoute.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import camelcaseKeys from 'camelcase-keys';
+import { Route } from '@tupaia/server-boilerplate';
+import { DatatrakWebRecentSurveyResponseRequest } from '@tupaia/types';
+
+export type RecentSurveyResponseRequest = Request<
+  DatatrakWebRecentSurveyResponseRequest.Params,
+  DatatrakWebRecentSurveyResponseRequest.ResBody,
+  DatatrakWebRecentSurveyResponseRequest.ReqBody,
+  DatatrakWebRecentSurveyResponseRequest.ReqQuery
+>;
+
+const ANSWER_COLUMNS = [
+  'text', 'question.code'
+]
+
+export class RecentSurveyResponseRoute extends Route<RecentSurveyResponseRequest> {
```

Comment:
This is pretty subjective so I don't mind if you want to keep it as is but I would prefer this to be named SurveyResponseRoute since it doesn't know (or need to know) that we are using it for recent survey responses.

Reply:
I can definitely change it. The only issue is that it looks a bit messy/confusing when we have stuff like:

```
SurveyResponsesRoute,
SurveyResponsesRequest,
SurveyResponseRoute,
SurveyResponseRequest,
```

Reply:
How about something like `SurveyResponseAnswersRoute`? Captures the use case _and_ differentiates it

Reply:
SingleSurveyResponseRoute sounds good to me 👍